### PR TITLE
Removes all remaining "netsync: false"s

### PR DESCRIPTION
**Pull request by Aexxie**

### CODE REVIEW

**Comment by ZoldorfTheWizard** [comment]
Emo is correct lights will go KERSPLOAHAHGETHAAA without netsync

**Comment by VasilisThePikachu** [comment]
This should probably not have a change log, it is
1. not something a player would notice
2. Noncontribs wont know what this mythical "netsync" is

**Comment by Aexxie** [comment]
One second I will fix the KAJSDHFLAOIKJUEDBGFLAEIKUJ

File: Resources/Prototypes/Entities/Clothing/Head/base_clothinghead.yml [hunk 1]
```diff
@@ -179,7 +178,6 @@
     equippedPrefix: off
   - type: ToggleableLightVisuals
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Clothing/Head/base_clothinghead.yml:182]: readd

File: Resources/Prototypes/Entities/Clothing/Head/base_clothinghead.yml [hunk 2]
```diff
@@ -60,7 +60,6 @@
     heldPrefix: off
   - type: ToggleableLightVisuals
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Clothing/Head/base_clothinghead.yml:63]: readd

File: Resources/Prototypes/Entities/Clothing/Head/hardhats.yml [hunk 3]
```diff
@@ -13,7 +13,6 @@
   - type: Clothing
     equippedPrefix: off
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Clothing/Head/hardhats.yml:16]: readd

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml [hunk 4]
```diff
@@ -41,7 +41,6 @@
     footstepSoundCollection:
       collection: FootstepClown
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:44]: readd

File: Resources/Prototypes/Entities/Effects/portal.yml [hunk 5]
```diff
@@ -32,7 +32,6 @@
     layers:
     - state: portal-red
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Effects/portal.yml:35]: readd

File: Resources/Prototypes/Entities/Effects/portal.yml [hunk 6]
```diff
@@ -45,7 +44,6 @@
     layers:
     - state: portal-blue
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Effects/portal.yml:48]: readd

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 7]
```diff
@@ -164,7 +164,6 @@
   - type: Sprite
     state: plushie_lamp
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:167]: readd

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 8]
```diff
@@ -920,7 +919,6 @@
     - type: UseDelay
       delay: 1.0
     - type: PointLight
-      netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:923]: readd

File: Resources/Prototypes/Entities/Objects/Misc/fluff_lights.yml [hunk 9]
```diff
@@ -46,7 +46,6 @@
     size: 20
     heldPrefix: off
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Misc/fluff_lights.yml:49]: readd

File: Resources/Prototypes/Entities/Objects/Misc/torch.yml [hunk 10]
```diff
@@ -40,7 +40,6 @@
       color: "#E25822"
       radius: 1.0
       energy: 5.0
-      netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Misc/torch.yml:43]: readd

File: Resources/Prototypes/Entities/Objects/Tools/flare.yml [hunk 11]
```diff
@@ -46,7 +46,6 @@
     color: "#FF8080"
     radius: 1.0
     energy: 9.0
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Tools/flare.yml:49]: readd

File: Resources/Prototypes/Entities/Objects/Tools/flashlights.yml [hunk 12]
```diff
@@ -63,7 +63,6 @@
   - type: Item
     sprite: Objects/Tools/flashlight.rsi
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Tools/flashlights.yml:66]: readd

File: Resources/Prototypes/Entities/Objects/Tools/glowstick.yml [hunk 13]
```diff
@@ -37,7 +37,6 @@
       color: "#00FF00"
       radius: 5
       energy: 0
-      netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Tools/glowstick.yml:40]: readd

File: Resources/Prototypes/Entities/Objects/Tools/lantern.yml [hunk 14]
```diff
@@ -41,7 +41,6 @@
       sprite: Objects/Tools/lantern.rsi
       heldPrefix: off
     - type: PointLight
-      netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Tools/lantern.yml:44]: readd

File: Resources/Prototypes/Entities/Objects/Tools/welders.yml [hunk 15]
```diff
@@ -53,7 +53,6 @@
         Heat: 8
         Blunt: -5
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Tools/welders.yml:56]: readd

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml [hunk 16]
```diff
@@ -98,7 +97,6 @@
   - type: UseDelay
     delay: 1.0
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml:101]: readd

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 17]
```diff
@@ -80,7 +80,6 @@
   components:
     - type: PointLight
       enabled: true
-      netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:83]: readd

File: Resources/Prototypes/Entities/Structures/Power/Generation/generators.yml [hunk 18]
```diff
@@ -100,7 +100,6 @@
     sound:
       path: /Audio/Ambience/Objects/hdd_buzz.ogg
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Structures/Power/Generation/generators.yml:103]: readd

File: Resources/Prototypes/Entities/Structures/Power/apc.yml [hunk 19]
```diff
@@ -13,7 +13,6 @@
     sound:
       path: /Audio/Ambience/Objects/hdd_buzz.ogg
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Structures/Power/apc.yml:16]: readd

File: Resources/Prototypes/Entities/Structures/Power/substation.yml [hunk 20]
```diff
@@ -114,7 +114,6 @@
     sound:
       path: /Audio/Ambience/Objects/buzzing.ogg
   - type: PointLight
-    netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Structures/Power/substation.yml:117]: readd

File: Resources/Prototypes/Entities/Structures/Specific/dragon.yml [hunk 21]
```diff
@@ -25,7 +25,6 @@
       color: "#366db5"
       radius: 10.0
       energy: 5.0
-      netsync: false
```
Comment (Aexxie) [inline Resources/Prototypes/Entities/Structures/Specific/dragon.yml:28]: readd

**Comment by Aexxie** [comment]
I did not mean to send a million notifications, only meant to keep track of what ive changed qwq

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml [hunk 22]
```diff
@@ -41,12 +41,11 @@
...
     enabled: true
     radius: 3
     energy: 2
     color: "#FFD800"
-
+    netsync: false
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:48]: skipped a line

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml [hunk 23]
```diff
@@ -41,12 +41,11 @@
...
 - type: entity
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:49]: skipped line

**Pull request merged**